client/src/components/Header.tsx

```diff
@@ -22,7 +22,7 @@ import LinkButton from './Atoms/LinkButton'
 const SupportBar = styled(Navbar)`
   background-color: ${Colors.ROSE3};
   height: 35px;
-  padding: 0;
+  padding: 0 10%;
```

Comment:
What's the thinking behind using a percentage here? I generally think it's safer to work with absolute values for padding

Reply:
It was to match the 80%, 10% on both sides